test-app/tests/integration/components/headless-form-test.gts

```diff
@@ -355,25 +379,35 @@ module('Integration Component headless-form', function (hooks) {
               <field.label>Last Name</field.label>
               <field.input data-test-last-name />
             </form.field>
+            <form.field @name="acceptTerms" as |field|>
+              <field.label>Terms accepted</field.label>
+              <field.checkbox data-test-terms />
+            </form.field>
             <button type="submit" data-test-submit>Submit</button>
           </HeadlessForm>
         </template>);
 
         assert.dom('input[data-test-first-name]').hasValue('Tony');
         assert.dom('input[data-test-last-name]').hasValue('Ward');
+        assert.dom('input[data-test-terms]').isNotChecked();
 
         await fillIn('input[data-test-first-name]', 'Nicole');
         await fillIn('input[data-test-last-name]', 'Chung');
+        await click('input[data-test-terms]');
         await click('[data-test-submit]');
 
         assert.deepEqual(
           data,
-          { firstName: 'Tony', lastName: 'Ward' },
+          { firstName: 'Tony', lastName: 'Ward', acceptTerms: false },
           'data is not mutated'
```

Comment:
Suggestion: `'data is not mutated'` -> 'original data is not mutated'

Reply:
applied :+1: